packages/tradershub/src/flows/RealAccountSIgnup/SignupWizard/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useCallback, useEffect } from 'react';
+import ReactModal from 'react-modal';
+import { useBreakpoint } from '@deriv/quill-design';
+import { StandaloneXmarkBoldIcon } from '@deriv/quill-icons';
+import { Text } from '@deriv-com/ui';
+import { ProgressBar } from '../../../components/ProgressBar';
+import { CUSTOM_STYLES } from '../../../helpers/signupModalHelpers';
+import { ACTION_TYPES, useSignupWizardContext } from '../../../providers/SignupWizardProvider';
+import WizardScreens from './WizardScreens';
+import './index.scss';
+
+const FORM_PROGRESS_STEPS = [
+    { isFilled: false, title: 'Account currency' },
+    { isFilled: false, title: 'Personal details' },
+    { isFilled: false, title: 'Address' },
+    { isFilled: false, title: 'Terms of use' },
+];
+
+/**
+ * @name SignupWizard
+ * @description The SignupWizard component is used to render the signup wizard modal.
+ * @example
+ * return (
+ *  <SignupWizard />
+ * );
+ */
+const SignupWizard = () => {
+    const { currentStep, dispatch, helpers, isWizardOpen, setIsWizardOpen } = useSignupWizardContext();
+    const { isMobile } = useBreakpoint();
+    useEffect(() => {
+        ReactModal.setAppElement('#v2_modal_root');
+    }, []);
+
+    const handleClose = useCallback(() => {
+        setIsWizardOpen(false);
+        dispatch({ type: ACTION_TYPES.RESET });
+        helpers.setStep(1);
+    }, [dispatch, helpers, setIsWizardOpen]);
+
+    return (
+        <ReactModal isOpen={isWizardOpen} onRequestClose={handleClose} shouldCloseOnOverlayClick style={CUSTOM_STYLES}>
+            <div className='bg-background-primary-base md:h-[717px] md:w-[1040px] h-screen w-screen md:rounded-800 flex overflow-hidden'>
```

Comment:
it's not fit in 678 px < width < 1040
```suggestion
            <div className='bg-background-primary-base md:h-[717px] md:max-w-[1040px] h-screen w-screen md:rounded-800 flex overflow-hidden'>
```